### PR TITLE
Feature/update

### PR DESCRIPTION
**Pull request by andreicek**

### CODE REVIEW

File: Frontend development/Susy/About Susy.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ _**This does NOT set box-sizing globally.**_
 There is a mixin for that which will be explained in the guide section.
 
 ### Other
-There are also [debug](http://susydocs.oddbird.net/en/latest/settings/#debug) and [custom options](http://susydocs.oddbird.net/en/latest/settings/#custom-support), which are not important for the purpose for this guide, and as such you can read about those in the documentation.
+There are also [debug](http://susy.readthedocs.io/settings/#debug) and [custom options](http://susy.readthedocs.io/settings/#custom-support), which are not important for the purpose for this guide, and as such you can read about those in the documentation.
```
Comment (DarkoKukovec) [inline Frontend development/Susy/About Susy.md]: We should update the docs to reflect this is Susy 2 specific. Same on the Guide page.

**Comment by andreicek** [comment]
@adrianbicanic please fix the comment above and merge before #7. The main part of this PR is reorganisation of the whole handbook.

**Comment by adrianbicanic** [comment]
@andreicek the guide is updated according to Darko's comment. Please merge after you check if everything is fine.

**Comment by andreicek** [comment]
This looks fine to me. @DarkoKukovec and @mjanjic01 please see if this is ok.

File: README.md [hunk 2]
```diff
@@ -1 +1,24 @@
-# frontend-handbook
\ No newline at end of file
+# frontend-handbook
```
Comment (DarkoKukovec) [inline README.md]: This todo list should probably be a private page

**Comment by andreicek** [comment]
@DarkoKukovec and @mjanjic01 take a look please!

File: Frontend development/Libraries.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+## Styling
+
+* **Susy**
+  * Susy is an a responsive customizable layout toolkit for Sass.
+  * More info with code examples is available [here.](http://susy.oddbird.net/)
+* **Normalize.css**
+  * A small CSS file that provides better cross-browser consistency in the default styling of HTML elements. A modern, HTML5 ready alternative to traditional CSS reset.
```
Comment (mjanjic01) [inline Frontend development/Libraries.md]: ...to **the** traditional...

Comment (dingo-d) [reply Frontend development/Libraries.md]: 'Susy is a responsive', not 'an a responsive...'

**Pull request merged**